{{cookiecutter.repository_name}}/.github/workflows/tests.yml

```diff
@@ -0,0 +1,67 @@
+name: Run Tests
+
+on: [push]
```

Comment:
I think we should add Pull request events here:

```
on:
  push:
    branches: [ '*' ]
  pull_request:
    branches: [ master ]
```

Reply:
sure